### PR TITLE
Patternlab/DP-15986: Update key message

### PR DESCRIPTION
**Pull request by avrilpearl**
## To review:
1. Visit page with key message solid in header (http://localhost:3000/?p=pages-marketing-campaign-as-solid). Confirm matches wireframe specs and lines up with BE (Drupal edit side).
2. Visit page with key message overlay in header (http://localhost:3000/?p=pages-marketing-campaign). Confirm matches wireframe specs and lines up with BE (Drupal edit side).

### Fixed
- (Patternlab) [Video]: Updated iframe video allow params to prevent console error. #758
### Changed
- (Patternlab) [MarketingCampaign]: Updated template so bottom key message lines up with footer. #758
- (Patternlab) [CampaignFeature]: Update campaign feature to use ma-container mixin gutters. #758
- (Patternlab) [KeyMessage]: Update key message component to pass color themes. Update to render with boxed callout and solid overlay callout.
### Added
- (Patternlab) [Button] DP-15986: Adds primary button theme variant of c-white. #758

### CODE REVIEW

File: patternlab/styleguide/source/_patterns/01-atoms/09-media/video.twig [hunk 1]
```diff
@@ -14,6 +14,6 @@
         {% include "@atoms/decorative-link.twig" %}
       </div>
     {% endif %}
-    <iframe width="{{ video.width }}" height="{{ video.height }}" src="{{ video.src }}" frameborder="0" allowfullscreen aria-label="{{ video.label }}"></iframe>
+    <iframe width="{{ video.width }}" height="{{ video.height }}" src="{{ video.src }}" frameborder="0" allowfullscreen allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" aria-label="{{ video.label }}"></iframe>
```
Comment (ygannett) [inline patternlab/styleguide/source/_patterns/01-atoms/09-media/video.twig]: I'd like to ensure video wouldn't play automatically. Can you change  `autoplay;` to `autoplay: 'none';` in the `allow` attribute value?

**Review by ygannett — CHANGES_REQUESTED** [review]
Please see my comment for `01-atoms/09-media/video.twig`.

File: patternlab/styleguide/source/_patterns/01-atoms/09-media/video.twig (hunk 1 above)
Comment (avrilpearl) [inline patternlab/styleguide/source/_patterns/01-atoms/09-media/video.twig]:
```suggestion
    <iframe width="{{ video.width }}" height="{{ video.height }}" src="{{ video.src }}" frameborder="0" allowfullscreen allow="accelerometer; fullscreen; gyroscope; picture-in-picture" aria-label="{{ video.label }}"></iframe>
```

Comment (avrilpearl) [reply patternlab/styleguide/source/_patterns/01-atoms/09-media/video.twig]: removed autoplay from the iframe per your comment

Comment (ygannett) [reply patternlab/styleguide/source/_patterns/01-atoms/09-media/video.twig]: Thank you.

**Review by ygannett — APPROVED** [review]

File: patternlab/styleguide/source/_patterns/03-organisms/by-template/key-message.twig [hunk 2]
```diff
@@ -1,46 +1,116 @@
...
+      }
+    </style>
+  {% elseif bgImage %}
+    <style>
+      #img{{ id }}.ma__key-message__inline-image {
+        background-image: url('{{ bgImage }}');
+      }
+    </style>
+  {% endif %}
+  {% if bgImage and positionTop %}
```
Comment (avrilpearl) [inline patternlab/styleguide/source/_patterns/03-organisms/by-template/key-message.twig:59]: Note: This is not ideal but the only way to set a margin based on the variable height of the overlay box's content.

**Pull request merged**